src/citrine/informatics/design_spaces.py

```diff
@@ -60,6 +60,12 @@ class ProductDesignSpace(Resource['ProductDesignSpace'], DesignSpace):
         serializable=False
     )
     active = properties.Boolean('active', default=True)
+    experimental = properties.Boolean("experimental", serializable=False, default=True)
```

Comment:
If `experimental_reasons` is to be populated when `experimental == True`, then it seems the default should be `False`. I imagine, however, that this default is proposed as a conservative choice during the transition to experimental annotations (and implicitly marking mature features as such by overriding this default).

Reply:
Yeah, this is a very short term conservative default.  As soon as the backend change goes out to all deployments, we won't be triggering this default anymore and I'd be inclined to remove it.

Reply:
>  As soon as the backend change goes out to all deployments, we won't be triggering this default anymore and I'd be inclined to remove it.

Yep. makes sense.